tracks/latest.toml

```diff
@@ -15,6 +15,7 @@ queries = [
     "../specs/aggregate.toml",
     "../specs/window_functions.toml",
     "../specs/like.toml",
+    "../specs/ilike.toml",
```

Comment:
If you add this to tthe `queries` block then it will only run the `setup` / `teardown` from `fixtures`. `fixtures` currently only creates part of the `uservisits` table.

So either the `ilike.toml` has to be adapted or moved to `full`.

We could also add the 2 new queries to `like.toml` and use the uservisits table. Would avoid having to setup another table in the nightly runs.

Reply:
Having loaded uservisits onto a local instance of cratedb, I perform the query:

```cr> select  "UserAgent"  from uservisits where "UserAgent" like '%iPhone%';
+-----------+
| UserAgent |
+-----------+
+-----------+
SELECT 0 rows in set (0.005 sec)```

if I replace like by **ilike** then I do get results... going to investigate why.